### ISSUE TITLE
WARNING: ev_ref is deprecated, use uv_ref

### ISSUE THREAD

**Issue by ghost** [mariano/node-db-mysql#77]
Using node 0.8.1 I get the above error message when executing db-mysql methods.

cu
Roman

**Comment by QuinoBLN** [comment]
Having the same behaviour here with v. 0.8.2.

cu
Quino

**Comment by efolio** [comment]
Me too on Node v0.8.4 : 
 "WARNING: ev_unref is deprecated, use uv_unref"

This might help on solving the issue: http://stackoverflow.com/questions/11755395/node-js-fixing-ev-ref-ev-unref-warning

**Comment by qraynaud** [comment]
Yes, this issue is fixed on trunk. Hope mariano will release soon.

**Comment by sharonjl** [comment]
I am still experiencing this issue.

**Comment by NachoSoto** [comment]
+1

**Comment by gecata83** [comment]
I'm having the same problem, maybe there is another modul for db-mysql?

**Comment by nabeelio** [comment]
Is this fix going to be released?

**Comment by jornare** [comment]
+1

**Comment by mattmcla** [comment]
+1

**Comment by qraynaud** [comment]
According to my perception, even if the maintainer often says otherwise on github, he is not maintaining node-db related packages. I proposed a patch months ago that he read & accepted but never commited. I saw another patch ready to roll out that never made it for the same reasons.

This could easily be made right if the author would accept to give rights on the repository to someone else that would do the job in the meantime (I believe he really intend to support those but is working on other projects and thus can't find the time to do both).

As of now, I only see one other solution : to fork the project. It is not very practical and you have to make people do the switch after that. That's not an awesome way of doing this.

**Comment by ghost** [comment]
+1 too.I am just deal with db-msyql.

**Comment by FaridSafi** [comment]
+1
